src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -106,7 +106,50 @@ private void updateHearing(HearingWrapper wrapper) throws UpdateCaseException, I
                 response.toString());
 
         hearingResponseUpdate(wrapper, response);
+    }
+
+    private HmcUpdateResponse createHmcUpdateResponse(HearingWrapper wrapper) throws InvalidMappingException {
+        HearingRequestPayload hearingPayload = buildHearingPayload(wrapper, referenceDataServiceHolder);
+        String hearingId = getHearingId(wrapper);
+        HmcUpdateResponse updateResponse;
+        String hearingRequestType;
+
+        try {
+            HearingGetResponse getResponse = hmcHearingApiService.getHearingRequest(hearingId);
```

Comment:
This wont work as hearing id will be null for create hearings, you need to use https://tools.hmcts.net/confluence/display/HMAN/hearings+Endpoints+Processing+and+Payloads not hearing endpoint. With the /hearings endpoint you can use the caseId then search for any that are awaiting listing